maat-court-data-api/src/main/java/gov/uk/courtdata/laastatus/builder/RepOrderUpdateMessageBuilder.java

```diff
@@ -81,39 +81,13 @@ private DefenceOrganisation mapDefenceOrganisation(CaseDetails caseDetails) {
     private Organisation findSolicitorDetails(SolicitorMAATDataEntity solicitorMAATDataEntity) {
 
         return Organisation.builder()
-                //TODO: to be enabled when CDA handled invalid data.
                 //.address(mapAddress(solicitorMAATDataEntity))
```

Comment:
if this is not required then could remove these two lines.

Reply:
Done